core/src/main/java/hudson/model/CharsetTaskListener.java

```diff
@@ -0,0 +1,11 @@
+package hudson.model;
```

Comment:
I don't like to use separate packages for things that definitively are related, prefer `jenkins` package to be used for (real) fresh new APIs. But can move, for sure if there's strong requirement here.